docker/php/conf.d/symfony.dev.ini

```diff
@@ -1,11 +1,14 @@
 apc.enable_cli = 1
 date.timezone = UTC
-session.auto_start = Off
-short_open_tag = Off
```

Comment:
They already are the defaults: https://github.com/php/php-src/blob/master/php.ini-development